docs/operating-scylla/procedures/cluster-management/rebuild-node.rst

```diff
@@ -2,18 +2,24 @@
 Rebuild a Node After Losing the Data Volume
 ============================================
 
-When running in Scylla on EC2, it is recommended to use i3 type instances, storing the data on fast, ephemeral SSD drives.
+When running in Scylla on EC2, it is recommended to use i4i and i3en type instances, storing the data on fast, ephemeral SSD drives.
 Stopping the node and starting it for whatever reason means the data on the node is lost.
```

Comment:
Usually, when we refer to `node` we mean the scylla node, i.e. the scylla process.
It would be more accurate to say "Stopping the **instance**", rather than **node**.